inspire-atom/src/main/java/org/fao/geonet/services/inspireatom/AtomSearch.java

```diff
@@ -200,7 +200,7 @@ public Element feeds(
         summary = "Get ATOM feeds",
         description = "")
     @GetMapping(
-        value = "/feeds",
+        value = "/feedsAsHtml",
```

Comment:
Would it be better to have one end point only and check the Accept header to return the proper response ?
or if we have specific path maybe `feeds.html`  or `feeds/html` could be more consistent with what is existing ?
![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/9e825c29-0115-4348-a1cc-d7c8aaff676d)

Reply:
Isn't that going to produce the same behaviour as before? see https://stackoverflow.com/a/24195263

Reply:
@jahow 
If you do it similar to the way you indicate in the link then this will work but create multiple io.swagger.v3.oas.annotations.Operation
Also, if ever you wanted to have json, or others content type, then it would end up being one api per type which could be more difficult to manage.

I believe it is better to use multiple "`produces`" and then simply parse the HTTP Accept Header.

This would also be inline with other api's like the following

https://github.com/geonetwork/core-geonetwork/blob/6c8c450ff6dd23babd1e22237e59dfad60065445/services/src/main/java/org/fao/geonet/api/records/MetadataApi.java#L143-L153